CruCentralCoast/Events/EventDetailsVC.swift

```diff
@@ -89,11 +96,14 @@ class EventDetailsVC: UIViewController {
         guard let facebookURL = event?.facebookUrl else { return }
         
         if let url = URL(string: facebookURL) {
+            self.facebookButton.backgroundColor = UIColor(red: 59, green: 89, blue: 153, alpha: 1)
```

Comment:
This should technically be an extension on UIColor. Probably named "facebookColor" or "facebookBlue" or something.

Reply:
good point!

Reply:
I actually do not need that line and forgot to remove it.

Reply:
But i still added the color to the xAssets folder!

Reply:
In addition to the Assets folder, make sure to add it to the UIColorExtensions.swift file